README.MD

```diff
@@ -80,8 +80,8 @@ A conditional value `{{{c?if_true||if_false}}}` will generate a function
 taking an optional parameter `?(c=false)` to define if `if_true` or `if_false`
 needs to be printed.
 
-Languages does not need to use the same labelled variables. The compiler
+Languages does not need to use the same labeled variables. The compiler
```

Comment:
Both are correct

Reply:
There were two other occurrences of 'labeled' in the file